doc/man5/x509v3_config.pod

```diff
@@ -180,7 +180,8 @@ L<ASN1_generate_nconf(3)> format.
 
 Examples:
 
- subjectAltName=email:copy,email:my@other.address,URI:http://my.url.here/
+ subjectAltName=email:my@other.address
+ subjectAltName=email:copy,URI:http://my.url.here/
```

Comment:
Is there a reason for this?

Reply:
It's not incorrect.  It fixes a linewrap complaint.  Having multiple email addresses in the SAN is confusing.  If you want I will revert that.

Reply:
I don't see this mentioned in #6473, so I assume you found this on your own. My `perldoc doc/man5/x509v3_config.pod` run does *not* linewrap this line, hence my surprise...

Reply:
Yes, I ran the lintian script equivalent given there by myself.

Reply:
> Having multiple email addresses in the SAN is confusing

Why is having multiple email addresses in the SAN confusing?